content/applications/sales/sales/send_quotations/pdf_quote_builder.rst

```diff
@@ -221,7 +221,10 @@ product PDF that was set to be visible inside the quote, can be viewed and print
 
 .. note::
    Download these :download:`PDF quote builder examples
-   <pdf_quote_builder/pdfquotebuilderexamples.zip>` for added reference.
+   <pdf_quote_builder/pdfquotebuilderexamples.zip>` and
+   :download:`Sample Quotation
+   <pdf_quote_builder/sample_quotation.pdf>`
+   for added reference.
```

Comment:
IMO sentence should
`Download these` **PDF quote builder examples** `or download` **sample quotation** `for add reference`.